.github/workflows/qa-automation.yml

```diff
@@ -0,0 +1,28 @@
+name: Add pull requests to project
+
+on:
+  pull_request:
+    types:
+      - review_requested
```

Comment:
You need to specify the branch, probably `development`

Reply:
You mean, only for the prs whose base is 'development'?

Reply:
fixed.